.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+name: Release
+
+permissions:
+  contents: write
+  id-token: write
```

Comment:
Do we need `id-token: write`? I don't see where we use OIDC token in the workflow.